### ISSUE TITLE
Column headers move/disappear when scrolling in V4

### ISSUE THREAD

**Issue by madiganz** [vaadin/vaadin-grid#1190]
### Description
When scrolling down, the column headers either temporarily disappear or fully disappear until user scrolls further.

### Expected outcome
Column headers should not move or disappear
![header](https://user-images.githubusercontent.com/18340029/35292087-46ed9924-0024-11e8-8c71-18fe9b56f009.JPG)

### Actual outcome
Column headers disappear
![noheader](https://user-images.githubusercontent.com/18340029/35293087-7b5e5308-0027-11e8-8935-9c921bb44d25.JPG)

### Browsers Affected

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE 11
- [ ] iOS Safari
- [ ] Android Chrome

**Comment by jouni** [comment]
I can also see this happening sometimes in v5, but only when my mouse cursor is on top of a `vaadin-checkbox`, like in the selection column.

**Comment by madiganz** [comment]
Very interesting. This seems to happen to me for all my tables. I tried to repro it in a jsbin and had the same problem, but then it started to work as normal. I am not sure why this is always happening for me. Could be something to do with custom styling?

**Comment by madiganz** [comment]
I just noticed that my body cells have padding. When I scroll with my mouse on the padding, I see this problem. When it is on vaadin-grid-cell-content, scrolling appears normal.

**Comment by jouni** [comment]
This also affects frozen columns, they jump similarly (as headers are basically just frozen rows).

Here’s a workaround (for v4):
```html
<dom-module id="grid-scroll-fix" theme-for="vaadin-grid">
  <template>
    <style>
      #scroller[scrolling] [part~="cell"] {
        pointer-events: none;
      }
    </style>
  </template>
</dom-module>
```

The `scrolling` attribute is set on an internal part (`#scroller`), so it’s slightly fragile perhaps. But it mitigates the issue almost completely (you only notice the jump when you start to scroll).

I’ll add this workaround directly to v5 theme.

**Comment by madiganz** [comment]
This works for me! Thank you.

**Comment by jouni** [comment]
Good to hear. It might be that @tomivirkki also fixed this in https://github.com/vaadin/vaadin-grid/commit/aaa2d03a8ea9d3c606575db785ef57990b807ffa (need to verify).

**Comment by jouni** [comment]
Just tested with @tomivirkki’s fix in master, and the workaround is no longer needed 🎉
Tomi, we probably want to backport the fix for 4.1 as well.

**Comment by tomivirkki** [comment]
Done. Tagged as 4.1.2
